examples/nfs/rook-nfs.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: ceph.rook.io/v1
 kind: CephNFS
 metadata:
   name: my-nfs
-  namespace: default
 spec:
   # For Ceph v15, the rados block is required. It is ignored for Ceph v16.
   rados:
-    # Ceph v16 always uses/expects "nfs-ganesha"
-    pool: nfs-ganesha
+    # some versins of Ceph v16 always uses/expects "nfs-ganesha" more recent
```

Comment:
versions.

Reply:
right, that is something that should be fixed in #3036

Reply:
correct the typo here `versins` to `versions`

Reply:
once rebased, after #3036 has been merged, this should have been addressed